source/community/reactnative/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# specific for windows script files
```

Comment:
Is this file required to be in GitHub?

Reply:
Have created a sample react native project with the latest react-native 0.62.0.

It includes a .gitattributes file. They mentioned, Its need for the windows configuration.